test/commands/apps/index.js

```diff
@@ -133,6 +140,20 @@ org-app-1   test-org@herokumanager.com
 example
 example-eu (eu)
 
+`)
+      })
+    })
+
+    it('shows locked apps', function () {
+      let mock = stubUserApps([example, euApp, lockedApp])
+      return apps.run({flags: {}, args: {}}).then(function () {
+        mock.done()
+        expect(cli.stderr).to.equal('')
+        expect(cli.stdout).to.equal(`=== foo@bar.com Apps
+example
+example-eu (eu)
+locked-app (locked)
```

Comment:
@RasPhilCo could you include an example of what it should look like if locked && not in US?